test/sharness/t0110-gateway.sh

```diff
@@ -63,6 +65,33 @@ test_expect_success "GET invalid path errors" '
   test_must_fail wget http://127.0.0.1:5002/12345
 '
 
+
+test_expect_success "GET webui forwards" '
+  curl -I http://127.0.0.1:5001/webui | head -c 18 > actual1
+  curl -I http://127.0.0.1:5001/webui/ | head -c 18 > actual2
+  echo "HTTP/1.1 302 Found" | head -c 18 > expected
+  echo "HTTP/1.1 301 Moved Permanently" | head -c 18 > also_ok
+  cat actual1
+  cat actual2
```

Comment:
maybe also divide by `webui` and `webui/` (may be easier to read/edit as two semi identical blocks)